docs/source/build_app/build-application.rst

```diff
@@ -132,6 +135,13 @@ Applications on Linux
             - libonedal_core.so, libonedal_sequential.so
             - libonedal_core.so, libonedal_thread.so
 
+     There are three available file names for dynamic linking: with major and minor binary versions, with major version or without versions.
```

Comment:
worth adding mention of semantic versioning https://semver.org/

And also recommendation to not use unversioned library for production use

Reply:
Mention and recommendation were added